appengine-java11/hello-world/README.md

```diff
@@ -0,0 +1,20 @@
+# Hello World App for Google App Engine Standard with Java 11
+
+This sample shows how to deploy an application to Google App Engine, using the
+`entrypoint` element in the [app.yaml](app.yaml) to start your application. The
+sample uses the `java` command is used to compile and execute the Java source code.
+
+## Setup
+See [Prerequisites](../README.md#Prerequisites).
+
+## Deploy to App Engine Standard
+
+```
+gcloud app deploy
```

Comment:
In one of the other samples it suggested using the mvn command line -- should we be pushing gcloud instead? Seems like we should be consistent.

Reply:
This is the most basic sample and doesn't use maven. Documentation will be used to explain the flexibility of the runtime.